isis/src/base/apps/shadow/shadow.xml

```diff
@@ -2,26 +2,28 @@
 <application name="shadow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
 
   <brief>
-    High accuracy hillshade with shadow casting
+    Create a high accuracy hillshade with shadow casting
   </brief>
 
   <description>
-    This program will create a shaded-relief cube from a DEM and match cube.<br/><br/>
+    This program, shadow, will create a shaded-relief cube from a digital
+    elevation model (DEM) and a MATCH cube.<br/><br/>
 
-    This program operates much like 'shade' but instead of an azimuth/elevation input, we use the
-      sun's position at the center of the given cube or at a given time and, by default, factor in
-      shadows cast by features.<br/><br/>
+    We use the sun's position at the center of the input cube or a user-defined
+    observation time. By default, we factor in shadows cast by features.
+    This program operates much like the ISIS3 'shade' program which requires azimuth/elevation as input.<br/><br/>

```

Comment:
"...which instead requires..." maybe to specifically delineate the difference between the two apps.